lib/datadog/tracing/sync_writer.rb

```diff
@@ -23,7 +23,7 @@ class SyncWriter
       # @param [Datadog::Transport::Traces::Transport] transport a custom transport instance.
       #   If provided, overrides `transport_options` and `agent_settings`.
       # @param [Hash<Symbol,Object>] transport_options options for the default transport instance.
-      # @param [Datadog::Core::Configuration::AgentSettingsResolver::AgentSettings] agent_settings agent options for
+      # @param [Datadog::Core::Configuration::AgentSettingsResolverCommon::AgentSettings] agent_settings agent options for
```

Comment:
TODO: Fix this comment. Should reference `Tracing::Configuration::AgentSettingsResolver::AgentSettings`